docs/06-Ecosystem.md

```diff
@@ -504,16 +504,17 @@ similar to what you use with MATLAB.
 
 ### Full-Stack Python
 
-Full-Stack Python refers to the design and deployment of web-based applications.  Using nothing *but* Python and supporting Python-base libraries, one can create complex web-apps which integrate into websites.  Python libraries are available for web frameworks, e.g., [Django](https://www.djangoproject.com/) and [Flask](https://flask.palletsprojects.com/), and front-end user interfaces, e.g., [Plotly Dash](https://plotly.com/dash/).
+Full-Stack Python refers to the design and deployment of web-based applications with significant server-side functionality.
```

Comment:
If they don't have significant server-side functionality, then they're just front-end, and not full-stack, right?

Reply:
Agreed.  Good catch.